.gitignore

```diff
@@ -1,2 +1,3 @@
 target/
 Cargo.lock
+.idea
```

Comment:
As this folder is specific to your choice of IDE, it should be added to your own local git configuration's `.gitignore`, not the one here.